metricbeat/module/mysql/query/_meta/fields.yml

```diff
@@ -0,0 +1,274 @@
+
+    - name: last_committed
+      type: long
+      description: >
+        The sequence number, or seqno, of the last committed transaction.
+
+    - name: local
+      type: group
+      description: >
+        Node specific Cluster status fields.
+      fields:
+        - name: bf_aborts
+          type: long
+          description: >
+            Total number of local transactions that were aborted by slave transactions while in execution.
+
+        - name: cert_failures
+          type: long
+          description: >
+            Total number of local transactions that failed certification test.
+
+        - name: commits
+          type: long
+          description: >
+            Total number of local transactions committed.
+
+        - name: recv
+          type: group
+          description: >
+            Node specific recv fields.
+          fields:
+            - name: queue
+              type: long
+              description: >
+                Current (instantaneous) length of the recv queue.
+
+            - name: queue_avg
+              type: double
+              description: >
+                Recv queue length averaged over interval since the last FLUSH STATUS command.
+                Values considerably larger than 0.0 mean that the node cannot apply write-sets as fast
+                as they are received and will generate a lot of replication throttling.
+
+            - name: queue_max
+              type: long
+              description: >
+                The maximum length of the recv queue since the last FLUSH STATUS command.
+
+            - name: queue_min
+              type: long
+              description: >
+                The minimum length of the recv queue since the last FLUSH STATUS command.
+
+        - name: replays
+          type: long
+          description: >
+            Total number of transaction replays due to asymmetric lock granularity.
+
+        - name: send
+          type: group
+          description: >
+            Node specific sent fields.
+          fields:
+            - name: queue
+              type: long
+              description: >
+                Current (instantaneous) length of the send queue.
+
+            - name: queue_avg
+              type: double
+              description: >
+                Send queue length averaged over time since the last FLUSH STATUS command.
+                Values considerably larger than 0.0 indicate replication throttling or network throughput issue.
+
+            - name: queue_max
+              type: long
+              description: >
+                The maximum length of the send queue since the last FLUSH STATUS command.
+
+            - name: queue_min
+              type: long
+              description: >
+                The minimum length of the send queue since the last FLUSH STATUS command.
+
+        - name: state
+          type: keyword
```

Comment:
I don't know what the value of this state look like. Should this state number be int or keyword?

Reply:
This has been removed and it will be included in a follow up PR. I... frankly don't remember now what was that value anyways :smile: